### ISSUE TITLE
after loss.backward()，the weight's grad is none

### ISSUE THREAD

**Issue by moonfighting** [qianjinhao/circle-loss#4]
When I calculate the loss and run loss.backward, 
I found all the weights in the model's grad are None ,
the weights' value can't be updated

the sample codes is like:
`
     features = model(input_data)

     loss = circle_loss(features, labels)

     loss.backward()

     for name, param in mode.named_parameters():

        print('name:', name,   "param_grad:', param.grad) # here all the param.grad is None
`
